.gitmodules

```diff
@@ -32,3 +32,7 @@
 [submodule "src/isa-l"]
 	path = src/isa-l
 	url = https://github.com/ceph/isa-l
+[submodule "src/zs"]
+	path = src/zs
+	url = https://github.com/SanDisk-Open-Source/zetascale.git
```

Comment:
we should make the ceph upstream a github.com/ceph clone of this repo so we can have any ceph-specific commits there.  i just created the clone.  we can set up a group to grant commit rights to whoever needs them